spec/system/company_value_system_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'rails_helper'
+
+RSpec.describe 'Company Value', type: :system do
```

Comment:
You are actually testing Company Values from Admin's dashboard, so I would consider more descriptive name (you will have to change it anyway, if there will be specs for Company Values from point of view of a User)
```suggestion
RSpec.describe 'Admin Company Values CRUD', type: :system do
```